packages/vuetify/src/composables/group.ts

```diff
@@ -56,17 +81,23 @@ export function useGroupItem (
     return group.isSelected(id)
   })
 
+  const selectedClass = computed(() => isSelected.value && (group.selectedClass.value ?? props.selectedClass))
+
   return {
     isSelected,
     toggle: () => group.select(id, !isSelected.value),
     select: (value: boolean) => group.select(id, value),
+    selectedClass,
+    value,
+    disabled,
   }
 }
 
 export function useGroup (
   props: GroupProps,
   injectKey: InjectionKey<GroupProvide>
 ) {
+  const isUnmounted = ref(false)
```

Comment:
why is it a ref?

Reply:
no reason other than muscle memory of defining a variable inside a composable